openupgrade_scripts/scripts/payment/15.0.2.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,183 @@
+---Models in module 'payment'---
+new model payment.refund.wizard [transient]
+# NOTHING TO DO: new model: https://github.com/odoo/odoo/commit/e0233a1010618450cf53b997c50964bed2bb1b1a
+
+---Fields in module 'payment'---
+payment      / account.payment          / source_payment_id (many2one)  : NEW relation: account.payment, isrelated: related, stored
+# NOTHING TO DO: it's related to source_transaction_id and the new refund feature
+
+payment      / account.payment.method.line / payment_acquirer_id (many2one): NEW relation: payment.acquirer, isfunction: function, stored
+# NOTHING TO DO
+
+payment      / payment.acquirer         / _order                        : _order is now 'module_state, state desc, sequence, name' ('module_state, state, sequence, name')
+payment      / payment.acquirer         / allow_tokenization (boolean)  : NEW
+payment      / payment.acquirer         / check_validity (boolean)      : DEL
+payment      / payment.acquirer         / inline_form_view_id (many2one): NEW relation: ir.ui.view
+payment      / payment.acquirer         / journal_id (many2one)         : not stored anymore
+payment      / payment.acquirer         / journal_id (many2one)         : now a function
```

Comment:
These 2 fields should go with the line about the model `account.payment.method.line`